packages/sealed_languages/CHANGELOG.md

```
@@ -1,3 +1,10 @@
+## 1.5.1
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)